File: Examples/DotNET/CSharp/Files/GetListFilesFromThirdPartyStorage.cs
using System;
using Com.Aspose.Storage.Api;
using Com.Aspose.Storage.Model;

namespace Files
{
    class GetListFilesFromThirdPartyStorage
    {
        public static void Run()
        {
            // ExStart:1           
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            string Path = "files";
            string storage = "MyStorageName"; 
            try
            {
                // Invoke Aspose.Storage Cloud SDK API to get list of files
                ResponseMessage apiResponse = storageApi.GetListFiles(Path, storage);
                if (apiResponse != null && apiResponse.Status.Equals("Ok"))
               {
                   Console.WriteLine(apiResponse.Message);
                   Console.ReadKey();
               }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/Node.js/Files/GetListFilesFromThirdPartyStorage.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var StorageApi = require('asposestoragecloud');
var configProps = require('../config/config.json');
var data_path = '../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var outFolder = configProps.out_folder;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);

try {	
	storageApi.GetListFiles(Path='files', storage='MyStorageName', function(responseMessage) {
		console.log('status:', responseMessage.status);
	});

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1